open_discussions/utils.py

```diff
@@ -186,3 +186,32 @@ def markdown_to_plain_text(markdown_str):
     """
     html_str = markdown2.markdown(markdown_str)
     return html_to_plain_text(html_str).strip()
+
+
+def prefetched_iterator(query, chunk_size=2000):
```

Comment:
Actually I guess the real thing here is that an iterator at all is probably not doing anything because `chunk_size` here is currently smaller than the indexing chunk size... I'm torn on whether to nix the iterator altogether because I should remove the usages and then it'll be unused entirely. Thoughts?

Reply:
My original and misconceived though was this was ideal for queries like `Post.objects.prefetch_related("article).iterator()` where it walks **all** records and `iterator()` causes `prefetch_related()` to be ignored.

Reply:
IMO if even if this isn't going to be used in this PR, I'd lean toward keeping it since its tested. We may want to build some kind of utils repo in the future and this could go in it